steampy/client.py

```diff
@@ -62,7 +62,7 @@ def set_proxies(self, proxies: dict) -> dict:
         if not isinstance(proxies, dict):
             raise TypeError(
                 'Proxy must be a dict. Example: '
-                '\{"http": "http://login:password@host:port"\, "https": "http://login:password@host:port"\}'
+                r'\{"http": "http://login:password@host:port"\, "https": "http://login:password@host:port"\}',
```

Comment:
Its not regex right?

Reply:
https://docs.astral.sh/ruff/rules/invalid-escape-sequence/